src/coins.cpp

```diff
@@ -443,6 +443,18 @@ double CCoinsViewCache::GetPriority(const CTransaction &tx, int nHeight) const
             dResult += coins->vout[txin.prevout.n].nValue * (nHeight-coins->nHeight);
         }
     }
+
+    // Joinsplits do not reveal any information about a note's amount or age.
+    // We need to ensure priority increases the longer a tx remains in the mempool, so we don't
+    // generate random values which would change on each invocation of this method.
+    // Instead, we use the anchor as a seed to generate the note value deterministically.
+    // Currently, we say that each JoinSplit has input notes of between 1 to 2 ZEC,
+    // and that the age of each note is half the age of the blockchain.
```

Comment:
@daira Just double-checking. If you were commenting on the extract shown on the ticket conversation page, the full comment is "We need to ensure priority increases the longer a tx remains in the mempool, so we don't generate random values".  Since for utxos, the priority is increased by: value * age (where value is constant).

We could treat each JoinSplit as a black box and say that since we don't have any information, we should treat them all equally.  So we just add 1,000,000,000,000 to the priority (which would be equal to 1 ZEC * 10000 or 17 days) for each JoinSplit.

On the other hand, we intuitively know (but of course cannot prove) that JoinSplits most likely have different input values, so we can at least try to account for this by introducing some variance.

I don't mind either way how we approach this.  We just need a big number.